contracts/adapters/PrtFeeSplitExtension.sol

```diff
@@ -0,0 +1,224 @@
+     * and update the snapshot.
+     */
+    function accrueFeesAndDistribute() public override onlyAllowedAccruer {
+        // Emits a FeeActualized event
+        streamingFeeModule.accrueFee(setToken);
+
+        uint256 totalFees = setToken.balanceOf(address(this));
+
+        uint256 operatorTake = totalFees.preciseMul(operatorFeeSplit);
+        uint256 prtTake = totalFees.sub(operatorTake);
+
+        if (operatorTake > 0) {
+            setToken.transfer(operatorFeeRecipient, operatorTake);
+        }
+
+        // Accrue PRT Staking Pool rewards and update snapshot
+        if (prtTake > 0) {
+            setToken.approve(address(prtStakingPool), prtTake);
+            prtStakingPool.accrue(prtTake);
+        }
+
+        emit PrtFeesDistributed(operatorFeeRecipient, address(prtStakingPool), operatorTake, prtTake);
+    }
+
+    /**
+     * @notice MUTUAL UPGRADE: Updates fee split between operator and PRT Staking Pool. Split defined in precise units (1% = 10^16).
+     * Does not accrue fees and snapshot PRT Staking Pool.
+     * @param _newFeeSplit Percent of fees in precise units (10^16 = 1%) sent to operator, (rest go to the PRT Staking Pool).
+     */
+    function updateFeeSplit(uint256 _newFeeSplit)
+        external
+        override
+        mutualUpgrade(manager.operator(), manager.methodologist())
+    {
+        require(_newFeeSplit <= PreciseUnitMath.preciseUnit(), "Fee must be less than 100%");
+        operatorFeeSplit = _newFeeSplit;
+        emit OperatorFeeSplitUpdated(_newFeeSplit);
+    }
+
+    /**
+     * @notice MUTUAL UPGRADE: Toggles the permission status of specified addresses to call the `accrueFeesAndDistribute()` function.
+     * @param _accruers An array of addresses whose accrue permission status is to be toggled.
+     * @param _statuses An array of booleans indicating the new accrue permission status for each corresponding address in `_accruers`.
+     */
+    function setAccruersStatus(
+        address[] memory _accruers,
+        bool[] memory _statuses
+    )
+        external
+        mutualUpgrade(manager.operator(), manager.methodologist())
+    {
+        _accruers.validatePairsWithArray(_statuses);
+        for (uint256 i = 0; i < _accruers.length; i++) {
+            _updateAccruersHistory(_accruers[i], _statuses[i]);
+            accrueAllowList[_accruers[i]] = _statuses[i];
+            emit AccruerStatusUpdated(_accruers[i], _statuses[i]);
+        }
+    }
+
+    /**
+     * @notice MUTUAL UPGRADE: Toggles whether or not anyone is allowed to call the `accrueFeesAndDistribute()` function.
+     * If set to true, it bypasses the accrueAllowList, allowing any address to call the `accrueFeesAndDistribute()` function.
+     * @param _status A boolean indicating if anyone can accrue.
+     */
+    function updateAnyoneAccrue(bool _status)
+        external
+        mutualUpgrade(manager.operator(), manager.methodologist())
```

Comment:
Same comment as above for `setAccruersStatus`

Reply:
updated to `onlyOperator` here [`8015c24`](https://github.com/IndexCoop/index-coop-smart-contracts/pull/176/commits/8015c24345302a42c4758bc1fe8ffbd41230ef00)